pom.xml

```diff
@@ -78,7 +78,8 @@
 		<!-- Other defaults -->
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<integration-test.pattern>**/*IntegrationTest*</integration-test.pattern>
-
+		<!-- exclude preview module code from sonar cloud analysis -->
+		<sonar.exclusions>**/spring/*SpringAutoConfiguration.*, **/spring/*SpringProperties.*</sonar.exclusions>
```

Comment:
Naive question: is there a reason for matching on the two classes rather than something more broad like `spring-cloud-previews/**`?

Reply:
+1 I think if we can get `spring-cloud-previews` in the pattern, it would be a more specific match.

Reply:
Good point. updated to use folder for exclusion.
This was a miss on my end: jacoco exclusions does not accept folders well, and forgot to change back when moving to sonar.exclusions.